### PR TITLE
whoami: add page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

File: pages/common/whoami.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# whoami
+
+> Display currently logged user
+
+- Display currently logged user id
```
Comment (waldyrious) [inline pages/common/whoami.md]: isn't the user ID a separate concept from user name? i.e. `whoami | id -u` should return the user ID, and `whoami` the user name.

**Comment by igorshubovych** [comment]
You are right. I updated it.

**Comment by waldyrious** [comment]
This looks good to me, but I'm a little concerned about the near (but not exact) repetition of the command description and the example description. Do you think one of them could be changed to use a different phrasing? For example, changing the top-level description to "Show the username of the current user" or something like that.

**Comment by igorshubovych** [comment]
I propose to merge it as is.

Scientific texts may contain tautology. The documentation does that too sometimes. You can later submit PR.

**Comment by igorshubovych** [comment]
@waldyrious i updated it

**Comment by waldyrious** [comment]
Thanks, LGTM

**Pull request merged**